### PR TITLE
Fix export Kefir 

### PR DESCRIPTION
**Pull request by cherifGsoul**
With this `Kefir` is exported instead of `kefir`.
for #4770

### CODE REVIEW

File: ecosystem.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export { default as debug } from "./es/can-debug";
 export { default as defineBackup } from "./es/can-define-backup";
 export { default as defineStream } from "./es/can-define-stream";
 export { default as defineStreamKefir } from "./es/can-define-stream-kefir";
-export { default as kefir } from "./es/can-kefir";
+export { default as Kefir } from "./es/can-kefir";
```
Comment (chasenlehara) [inline ecosystem.js]: This is a breaking change. We should export both for now.

Comment (cherifGsoul) [reply ecosystem.js]: Fixed

Comment (chasenlehara) [reply ecosystem.js]: Does this work?

```js
export { default as kefir, default as Kefir } from "./es/can-kefir";
```

Comment (cherifGsoul) [reply ecosystem.js]: I'll check it.

**Pull request merged**